src/Table/Table.js

```diff
@@ -123,27 +121,10 @@ export default class Table extends React.Component {
       newState.viewportHeight =
         props.windowRatio * DOMUtil.getViewportHeight() - headerHeight;
       // Calculated scroll container height
-      let scrollContainerHeight =
```

Comment:
Don't need scroll container height anymore, it's going to just be as long as it needs to be